test/e2e-tests.sh

```diff
@@ -120,6 +120,8 @@ kubectl -n "${SYSTEM_NAMESPACE}" patch configmap/config-leader-election --type=m
 add_trap "kubectl get cm config-leader-election -n ${SYSTEM_NAMESPACE} -oyaml | sed '/.*enabledComponents.*/d' | kubectl replace -f -" SIGKILL SIGTERM SIGQUIT
 # Delete HPA to stabilize HA tests
 kubectl -n "${SYSTEM_NAMESPACE}" delete hpa activator
+# Make sure all pods run in leader-elected mode.
+kubectl delete pods --all -n "${SYSTEM_NAMESPACE}"
```

Comment:
Hmm. Now I wonder if logs will be available for a dump

Reply:
I think in order to get proper logs in the event of failure, we'll need to run in leader-elected mode from the beginning of the test suites

Reply:
`kail` should pick them up as it goes.

Reply:
OK. Let's see what happens in CI.

Reply:
Yeah kail should tail new pods.

Reply:
I'm more worried about the old ones that are stopped. Is this covered too?